psychopy/hardware/speaker.py

```diff
@@ -181,6 +230,11 @@ def createStream(self):
                 "Failed to setup a PsychToolBox audio stream for device %(DeviceName)s "
                 "(%(DeviceIndex)s)." % self.profile
             )
+
+        logging.warning(
```

Comment:
Shouldn't this be `info` or `debug`?

Reply:
The same applies to alert - if the message is just "Creates stream for speaker device" I don't think we should be alerting them to it as if something's gone wrong

Reply:
Yes, that should be a lower log level. Changing it.